toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -787,16 +787,28 @@ func addGroups(installChroot *safechroot.Chroot, groups []configuration.Group) (
 }
 
 func addUsers(installChroot *safechroot.Chroot, users []configuration.User) (err error) {
+	const (
+		squashErrors = false
+	)
+
+	rootUserAdded := false
+
 	for _, user := range users {
 		logger.Log.Infof("Adding user (%s)", user.Name)
 		ReportActionf("Adding user: %s", user.Name)
 
-		var homeDir string
+		var (
+			homeDir string
+			isRoot  bool
+		)
 
-		homeDir, err = createUserWithPassword(installChroot, user)
+		homeDir, isRoot, err = createUserWithPassword(installChroot, user)
```

Comment:
Would using `rootUserAdded` in place of `isRoot` work? It feels like we're adding an unnecessary variable and an `if` statement.

Reply:
`rootUserAdded` holds whether the root user was specified in any of the imageconfig User entries. If we were to use `rootUserAdded` in place of `isRoot`, we would only track whether the last user entry was root.